pykrige/core.py

```diff
@@ -194,29 +191,70 @@ def _adjust_for_anisotropy(X, center, scaling, angle):
     return X_adj
 
 
-def initialize_variogram_model(x, y, z, variogram_model, variogram_model_parameters,
-                               variogram_function, nlags, weight, coordinates_type):
-    """Initializes the variogram model for kriging according
-    to user specifications or to defaults"""
+def initialize_variogram_model(X, y, variogram_model, variogram_model_parameters, variogram_function,
+                               nlags, weight, coordinates_type):
+    """Initializes the variogram model for kriging. If user does not specify parameters,
+    calls automatic variogram estimation routine.
 
-    x1, x2 = np.meshgrid(x, x, sparse=True)
-    y1, y2 = np.meshgrid(y, y, sparse=True)
-    z1, z2 = np.meshgrid(z, z, sparse=True)
-    dz = z1 - z2
+    Parameters
+    ----------
+    X: ndarray
+        float array [n_samples, n_dim], the input array of coordinates
+    y: ndarray
+        float array [n_samples], the input array of values to be kriged
+    variogram_model: str/unicode
```

Comment:
Yeah, probably fair... my IDE was just complaining about it so I made the change to make the yellow highlighting go away